### PR TITLE
#163 - Add Lorenz Model

### LINKED ISSUE #163: Add Lorenz model
https://en.wikipedia.org/wiki/Lorenz_system

### PR DESCRIPTION
**Pull request by SebastianGuadalupe**
Closes #163.

### CODE REVIEW

File: models/Lorenz/lorenz.jl [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#=
+Lorenz model from
+https://en.wikipedia.org/wiki/Lorenz_system
+=#
+using TaylorIntegration
+using TaylorModels: @taylorize
+using Reachability
+using MathematicalSystems
+using LazySets
```
Comment (schillic) [inline models/Lorenz/lorenz.jl]: Maybe merge these lines.

Comment (mforets) [reply models/Lorenz/lorenz.jl]: alright, i also kept it to the minimum number of packages that need to be loaded ATM

File: models/Lorenz/lorenz.jl [hunk 2]
```diff
@@ -2,11 +2,8 @@
 Lorenz model from
 https://en.wikipedia.org/wiki/Lorenz_system
 =#
-using TaylorIntegration
-using TaylorModels: @taylorize
-using Reachability
-using MathematicalSystems
-using LazySets
+using Reachability, MathematicalSystems, TaylorIntegration
+using Reachability: solve
```
Comment (schillic) [inline models/Lorenz/lorenz.jl]: `Reachability` exports `solve`

Comment (mforets) [reply models/Lorenz/lorenz.jl]: `TaylorIntegration` also exports `solve`, and if there are conflicts like this Julia usually prints a warning (or refuses to run the function).

Comment (mforets) [reply models/Lorenz/lorenz.jl]: however, with

``julia
using Reachability, MathematicalSystems, TaylorIntegration
```
The script runs without complaining about `solve` (in julia v1.2 -- iirc the behavior used to be different in previous versions). So let me remove line 6 in another pr..

Comment (schillic) [reply models/Lorenz/lorenz.jl]: This is a contradiction. Either both packages `export` the same symbol, and then there must be a conflict, or they do not.

Comment (schillic) [reply models/Lorenz/lorenz.jl]: > TaylorIntegration also exports solve

I think this is not true (maybe they changed this).

```julia
using TaylorIntegration
julia> using TaylorIntegration

help?> solve
search:

Couldn't find solve
Perhaps you meant Some, sort, sort!, isone, size, sleep, split, else, Core, Colon, foldl, foldr, one or isodd
  No documentation found.

  Binding solve does not exist.

julia> solve
ERROR: UndefVarError: solve not defined
```

**Pull request merged**